docs/book/src/introduction/installation.md

```diff
@@ -2,35 +2,132 @@
 
 The _Sway toolchain_ is sufficient to compile Sway smart contracts. Otherwise, note that if you want to run Sway smart contracts (e.g. for testing), a Fuel Core full node is required, which is packaged together with the _Sway toolchain_ together as the _Fuel toolchain_.
 
-## Installing from Pre-compiled Binaries
+## Install via Pre-compiled Binaries (Recommended)
 
-Pre-compiled release binaries for Linux and macOS are available for the Sway toolchain. Native Windows is currently unsupported ([tracking issue for Windows support](https://github.com/FuelLabs/sway/issues/1526)). Windows Subsystem for Linux should work but is not officially supported.
+Installing via pre-compiled release binaries is the recommended way to get up and running with the Sway toolchain. Pre-compiled binaries for Linux and macOS are available. Native Windows is currently unsupported ([tracking issue for Windows support](https://github.com/FuelLabs/sway/issues/1526)). Windows Subsystem for Linux should work but is not officially supported.
 
 [`fuelup`](https://github.com/FuelLabs/fuelup) is the equivalent of Rust's `rustup` for the Fuel toolchain. It enables easily downloading binary releases of the Fuel toolchain.
 
-Start by installing `fuelup` with:
-
-```sh
-curl --proto '=https' --tlsv1.2 -sSf \
-    https://fuellabs.github.io/fuelup/fuelup-init.sh | sh
-```
-
-`fuelup-init` will ask for permission to add `~/.fuelup/bin` to your PATH. Otherwise, you can also pass `--no-modify-path` so that `fuelup-init` does not modify your PATH:
-
-```sh
-curl --proto '=https' --tlsv1.2 -sSf \
-    https://fuellabs.github.io/fuelup/fuelup-init.sh | sh -s -- --no-modify-path
-```
-
-Once `fuelup` is installed, `fuelup-init` automatically runs the command below
-
-```sh
-fuelup toolchain install latest
-```
-
-to install the latest Fuel toolchain.
-
-You can run the same command at a later time to update the toolchain.
+1. Start by installing `fuelup` with the following command:
+
+   ```sh
+   curl --proto '=https' --tlsv1.2 -sSf \
+   https://fuellabs.github.io/fuelup/fuelup-init.sh | sh
+   ```
+
+   `fuelup-init` will ask for permission to add `~/.fuelup/bin` to your PATH. Otherwise, you can also pass `--no-modify-path` so that `fuelup-init` does not modify your PATH:
+
+   ```sh
+   curl --proto '=https' --tlsv1.2 -sSf \
+   https://fuellabs.github.io/fuelup/fuelup-init.sh | sh -s -- --no-modify-path
+   ```
+
+2. Once `fuelup` is installed, run the following command to install the latest Fuel toolchain:
```

Comment:
Ah I see why you asked about `fuelup-init`! You actually run `fuelup-init` in the first step here:

```sh
curl --proto '=https' --tlsv1.2 -sSf \
   https://fuellabs.github.io/fuelup/fuelup-init.sh | sh -s -- --no-modify-path
```
and that does everything for you, including the [toolchain installation step](https://github.com/FuelLabs/fuelup/blob/bd4855bfbca5df0886ab0f9966e7fd2044b89ad4/fuelup-init.sh#L130) - `fuelup-init` invokes `fuelup` to do the installation for you by default for convenience if you didn't provide `--skip-toolchain-installation`, so maybe the original documentation wasn't clear enough about that.

I'd suggesting reverting back to the original line or some variation that keeps the original meaning here, and we'll include the missing info above.

Reply:
Ok cool thanks. I'm not clear on if the user has to run `fuelup-init`, or if the users has to run `fuelup toolchain install latest` or neither and these get called automatically once you install fuelup? @bingcicle

Reply:
> curl --proto '=https' --tlsv1.2 -sSf \
   https://fuellabs.github.io/fuelup/fuelup-init.sh | sh
   
   this command invokes `fuelup-init`, then

> fuelup toolchain install latest

this is invoked by `fuelup-init` automatically once `fuelup` is done installing. So yes, it is automatic :)